### PR TITLE
#4306 Wider actionable space for buttons in navbar

### LINKED ISSUE #4306: Wider actionable space for buttons in navbar
Right now, the user needs to precisely hit the icon on kodadot.
To make this easier, let's make this space wider around the icon, at least 20px on each side.
From top and bottom make it the same as rest buttons in navbar

ht @vlad 
 
https://user-images.githubusercontent.com/5887929/201425210-ec580f66-8b23-4df5-a913-4f585d120015.mov

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #4306
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

**Comment by yangwao** [comment]
from left side still missing actionable space 
https://user-images.githubusercontent.com/5887929/201522024-89d5ab1e-6abe-4e0d-b7e2-e633713ebbc8.mov

**Comment by yangwao** [comment]
same missing from left on both logged and wallet
https://user-images.githubusercontent.com/5887929/201522071-802ff71d-3288-4d8b-9410-2e44340ab820.mov

**Comment by prachi00** [comment]
> same missing from left on both logged and wallet https://user-images.githubusercontent.com/5887929/201522071-802ff71d-3288-4d8b-9410-2e44340ab820.mov

fixed

File: styles/layouts/_main-navigation.scss [hunk 1]
```diff
@@ -179,6 +179,15 @@
...
 
 	#NavProfile {
+    .dropdown-trigger {
+      height: 40px;
+      width: 40px;
+      cursor: pointer;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
```
Comment (roiLeo) [inline styles/layouts/_main-navigation.scss:189]: pather

**Review by roiLeo — CHANGES_REQUESTED** [review]
I would love to see a solution without custom css rule..

Small tip:
- you can use `navbar-link is-arrowless` class on dropdown and it will do the work


spacing issue when not logged in:
Screenshot 2022-11-15 at 09-19-41 KodaDot - Kusama NFT Market Explorer

Navbar end has whitespace (burger menu)
Capture d’écran 2022-11-15 à 9 22 00 AM

File: styles/layouts/_main-navigation.scss (hunk 1 above)
Comment (prachi00) [reply styles/layouts/_main-navigation.scss]: so doing this shows the cursor pointer change but it doesnt open the dropdown sometimes, not sure why

**Comment by yangwao** [comment]
Closing in favour of 
- #4335

**Pull request closed without merging**